scripts/smoke-check.sh

```
@@ -0,0 +1,75 @@
+    --base-url) BASE_URL="$2"; shift ;;
+    -h|--help)
+      echo "Usage: bash scripts/smoke-check.sh [--base-url URL]"; exit 0 ;;
+  esac
+  shift
+done
+
+jq_ok=false
+if command -v jq >/dev/null 2>&1; then jq_ok=true; fi
+
+check_endpoint() {
+  local path="$1"; local name="$2"; local required="$3"
```

Shellcheck (0.10.0)

[warning] required appears unused. Verify use (or export if used externally).

(SC2034)